src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ErrorConfiguration.kt

```diff
@@ -22,7 +22,11 @@ enum class Code {
   CONDITIONAL_FIELD_MUST_BE_SET,
   CANNOT_BE_EMPTY,
   CANNOT_BE_NEGATIVE_OR_ZERO,
-  INVALID_SERVICE_CATEGORY_FOR_CONTRACT
+  INVALID_SERVICE_CATEGORY_FOR_CONTRACT,
+  ADDRESS_MUST_BE_PROVIDED,
+  INVALID_NPS_OFFICE_CODE,
+  INVALID_ADDRESS_FIELDS_COUNT,
+  INVALID_POST_CODE,
```

Comment:
It would be good to keep this `Code` ... i.e. type of error list tiny -- the errors are already scoped to the field, so we don't need new codes, right?

What do you think about using `INVALID_FORMAT` instead of `INVALID_<specific_field>`?

Reply:
I thought the caller would want to know why it's invalid. For IN_PERSON_MEETING_OTHER it could be for the following reasons:

1. The delivery address is empty or not provided
2. The delivery address has incorrect number of lines
3. The delivery address has an invalid post code

We can perhaps use the same code as 'INVALID_FORMAT' but it would help to also include a message along with that; but that would mean changing the FieldError object to accept an optional message field. 
How do you feel about that?

Reply:
Sounds good! Allowing for the field, the kind of problem, the message feels 👍 to me

Could still be serialised to `{"field": "...", "error": "$code: $message"}` but doesn't increase coupling to specific fields